brian2/codegen/runtime/numpy_rt/templates/spatialstateupdate.py_

```diff
@@ -26,7 +27,7 @@ _vectorisation_idx = 1
 _vectorisation_idx = LazyArange(N)
 {{vector_code|autoindent}}
 
-_v_previous = {{v}}.copy()
+{{_v_previous}} = {{v}}.copy()
```

Comment:
It's doesn't matter much here, but this will not store the copy in the reserved memory but instead overwrite the variable name. This means that e.g. during debugging, a user looking at the value of `neuron._v_previous` would not see the values set here. I'd therefore replace this with `{{_v_previous}}[:] = {{v}}`